pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -34,7 +34,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.21.2"
+	CustomResourceDefinitionSchemaVersion = "1.21.3"
```

Comment:
For this, I just ran check-compat-tables.h and did what it said without thought:

```
$ Documentation/check-crd-compat-ta
ble.sh v1.8
Current version for branch v1.8 should be 1.21.3, not 1.21.2, please run the following command to fix it:
git checkout origin/v1.8 && \
sed -i 's+1.21.2+1.21.3+' pkg/k8s/apis/cilium.io/v2/register.go
```

But I think this always increments for every release..? Is that intended @aanm ?

If not, how does someone actually know if the CRD changes? Are there specific file paths we are supposed to pay attention to?

Reply:
I don't think we want to increment the version. Last release I had the same issue and Andre mentioned to ignore it for now until it's fixed.

Reply:
Yeah if CRDs were not changed then we shouldn't increment this version